tensorbay/opendataset/HKD/loader.py

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env python3
+#
+# Copyright 2022 Graviti. Licensed under MIT Licence.
+#
+# pylint: disable=invalid-name
+
+"""Dataloader for Finger Tapping Hand Keypoints Dataset(HKD)."""
+import csv
+import os
+
+from tensorbay.dataset import Data, Dataset
+from tensorbay.label import LabeledKeypoints2D
+from tensorbay.utility import chunked
+
+DATASET_NAME = "HKD"
+_SEGMENT_INFO = {
+    "subject1": (
+        "subject1_fingercount_2D_Annotations_cropped.csv",
+        "subject1_fingercount_cropframe_{}.jpg",
+    ),
+    "subject2": (
+        "subject2_fingercount_2D_Annotations_cropped.csv",
+        "subject2_fingercount_cropframe_{}.jpg",
+    ),
+    "subject3": (
+        "fingerappose_subject3_2D_Annotations_cropped.csv",
+        "fingerappose_subject3_cropframe_{}.jpg",
+    ),
+    "subject4": ("subject4_2D_Annotations_cropped.csv", "subject4_cropframe_{}.jpg"),
+}
+
+
+def HKD(path: str) -> Dataset:
+    """`HKD <http://vlm1.uta.edu/~srujana/HandPoseDataset/HKD_Cropped/>`_ dataset.
+
+    The file structure should be like::
+
+        <path>
+            AnnotatedData_subject1/
+                CropImages/
+                    subject1_fingercount_cropframe_2.jpg
+                    subject1_fingercount_cropframe_3.jpg
+                    ...
+                    subject1_fingercount_cropframe_210.jpg
+                subject1_fingercount_2D_Annotations_cropped.csv
+
+            AnnotatedData_subject2/
+                CropImages/
+                    subject2_fingercount_cropframe_2.jpg
+                    subject2_fingercount_cropframe_3.jpg
+                    ...
+                    subject2_fingercount_cropframe_207.jpg
+                subject2_fingercount_2D_Annotations_cropped.csv
+
+            AnnotatedData_subject3/
+                CropImages/
+                    fingerappose_subject3_cropframe_2.jpg
+                    fingerappose_subject3_cropframe_3.jpg
+                    ...
+                    fingerappose_subject3_cropframe_235.jpg
+                fingerappose_subject3_2D_Annotations_cropped.csv
+
+            AnnotatedData_subject4/
+                CropImages/
+                    subject4_cropframe_2.jpg
+                    subject4_cropframe_3.jpg
+                    ...
+                    subject4_cropframe_147.jpg
+                subject4_2D_Annotations_cropped.csv
+
```

Comment:
too many blank lines

Reply:
should not blank lines make the codes' structure seems more clear?

Reply:
do not need to add 3 lines here

Reply:
my bad, didn't notice that some codes had been removed from here.